community/kernel/src/main/java/org/neo4j/kernel/impl/transaction/TransactionCounters.java

```diff
@@ -19,72 +19,33 @@
  */
 package org.neo4j.kernel.impl.transaction;
 
-import java.util.concurrent.atomic.AtomicLong;
-
-public class TransactionCounters implements TransactionMonitor
+public interface TransactionCounters
```

Comment:
do we really need to introduce this interface? what will be the other possible implementation?